### ISSUE TITLE
Swamphouse Reward impossible check

### ISSUE THREAD

**Issue by G4M3RL1F3** [LittlestCube/MMRecompRando#9]
TLDR: `Swamphouse Reward` check can be impossible if a `Swamp Skulltula Token` is randomized there.

I had a solo seed where every location has been checked with the exception of `Swamphouse Reward`, which requires all 30 `Swamp Skulltula Tokens`. However, I only had 29. According to the spoiler log, `Swamphouse Reward` had my 30th, making this check impossible.

My situation might not make it look like it's really that important to fix, but I can think of syncs where this situation might bring confusion as to why a player wouldn't be able to check this location. I had to check the spoiler log to make sure I didn't do anything wrong or if there was a generation issue. Really, this fix would only help avoid confusion for other people.

**Comment by LittlestCube** [comment]
Nope, it was a logic bug 🥴 just fixed.
